### PR TITLE
Update example create dev env script

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

File: scripts/create-developer-env.sh [hunk 1]
```diff
@@ -23,30 +23,19 @@ cmd "source ${SPACK_MANAGER}/start.sh && spack-start"
...
-# quick-develop --name --exawind --spec $SPEC hypre@develop
-# still missing view selection for this
 
-# updated to use the new --name flag
 cmd "spack manager create-env --name exawind --spec '${SPEC}'"
 cmd "spack env activate -d ${SPACK_MANAGER}/environments/exawind"
-# if you do develop first then it will now auto-blacklist those specs for you
-cmd "spack manager external ${SNAPSHOT_DIR} -v ${VIEW} --blacklist ${BLACKLIST}"
 cmd "${DEV_COMMAND}"
+cmd "spack manager external ${SNAPSHOT_DIR} --latest"
```
Comment (psakievich) [inline scripts/create-developer-env.sh]:
```suggestion
cmd "spack manager external --latest"
```

Comment (psakievich) [reply scripts/create-developer-env.sh]: Not defined in the script anymore. and not needed either

File: scripts/create-developer-env.sh [hunk 2]
```diff
@@ -23,30 +23,19 @@ cmd "source ${SPACK_MANAGER}/start.sh && spack-start"
...
   DEV_COMMAND='spack manager develop nalu-wind@master; spack manager develop hypre@develop'
-  VIEW="${COMPILER}"
 fi
-# all of these commands will eventually be combined into
-# quick-develop --name --exawind --spec $SPEC hypre@develop
-# still missing view selection for this
 
-# updated to use the new --name flag
 cmd "spack manager create-env --name exawind --spec '${SPEC}'"
 cmd "spack env activate -d ${SPACK_MANAGER}/environments/exawind"
```
Comment (psakievich) [inline scripts/create-developer-env.sh:36]: How do you feel about combining these with `quick-create`?

**Review by psakievich — CHANGES_REQUESTED** [review]

File: scripts/create-developer-env.sh (hunk 2 above)
Comment (jrood-nrel) [reply scripts/create-developer-env.sh:36]: I prefer the explicit commands.

Comment (psakievich) [reply scripts/create-developer-env.sh:36]: okay that's fine.

**Review by psakievich — APPROVED** [review]

**Pull request merged**